pkg/controller/memberstatus/memberstatus_controller_test.go

```diff
@@ -39,6 +42,25 @@ func TestNoMemberStatusFound(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, reconcile.Result{}, res)
 	})
+
+	t.Run("No memberstatus resource found - right name but not found", func(t *testing.T) {
+		// given
+		expectedErrMsg := "get failed"
+		requestName := defaultMemberStatusName
+		getHostClusterFunc := newGetHostClusterReady
+		reconciler, req, fakeClient := prepareReconcile(t, requestName, getHostClusterFunc)
+		fakeClient.MockGet = func(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
+			return fmt.Errorf(expectedErrMsg)
+		}
+
+		// when
+		res, err := reconciler.Reconcile(req)
+
+		// then - there should not be any error, the controller should only log that the resource was not found
```

Comment:
Need to fix the comment.

Reply:
Nice catch!

Fixed in c4d725ad046b4bf1e2f175f1bea36aa148d68ec3